crate/operator/create.py

```diff
@@ -128,7 +128,7 @@ async def create_debug_volume(
     name: str,
     labels: LabelType,
     logger: logging.Logger,
-) -> Tuple[V1PersistentVolume, V1PersistentVolumeClaim]:
+) -> None:
```

Comment:
Note: everywhere where the return type has been changed to `None`, the returned values have not been used for anything.